app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title = '')
+    base_title = 'Ruby on Rails Tutorial Sample App'
+    if page_title.empty?
```

Comment:
chỗ này mình viết toán tử 3 ngôi cho gọn được k e

Reply:
dạ e fix r a